homework/week1/2025320048-yujuyeon/[Homework][Week1] 1027번.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+typedef struct {
+    int h;
+    int num;
+} bldg;
+
+int left(bldg b[], int idx) {
+    double incli = 0;
+    double min_incli = 1000000001;
+    int renew = 0;
+    for (int i = 1; idx - i >= 0; i++) {
+        incli = ((double)(b[idx].h - b[idx - i].h)) / i; //연산 과정에서의 자료형이 중요함... *명시적 형 변환
```

Comment:
변수는 선언되는 블록은 가급적 안쪽이 좋습니다.
```suggestion
    double min_incli = 1000000001;
    int renew = 0;
    for (int i = 1; idx - i >= 0; i++) {
        double incli;
        incli = ((double)(b[idx].h - b[idx - i].h)) / i; //연산 과정에서의 자료형이 중요함... *명시적 형 변환
```